examples/scripts/utils.py

```diff
@@ -52,6 +54,12 @@ def make_tar(
     with tarfile.open(tarball_path, tarball_write_mode) as f:
         for dir in source_dir:
             dir = Path(dir)
-            f.add(dir, arcname=dir.name)
+            if os.path.isdir(dir):
+                for dirpath, dirnames, filenames in os.walk(dir):
+                    for name in filenames:
+                        name = Path(os.path.join(dirpath, name))
```

Comment:
`Path()` objects can also join paths in a cross-platform way without needing to use `os.path.join`:

```suggestion
                        name = Path(dirpath, name)
```